src/main.ts

```diff
@@ -117,6 +117,7 @@ module.exports = function (argv: string[]): void {
 		.option('--allow-star-activation', 'Allow using * in activation events')
 		.option('--allow-missing-repository', 'Allow missing a repository URL in package.json')
 		.option('--skip-license', 'Allow packaging without license file')
+		.option('--sign', 'Script to sign the VSIX package. VSIX manifest will be passed as an argument. Generates <package-name>.signature.zip alongside the VSIX package.')
```

Comment:
I suggest we change the language from "script" to "tool" and simplify the interface by passing two arguments to it:

```suggestion
		.option('--signtool', 'Path to the VSIX signing tool. Will be invoked with two arguments: `SIGNTOOL <path/to/extension.vsix> <path/to/output-signature.zip>`.')
```